.rultor.yml

```diff
@@ -12,11 +12,12 @@ assets:
   s3cfg: yegor256/home#assets/s3cfg
 merge:
   script: |-
+    pdd --source=$(pwd) --verbose --file=/dev/null --exclude=qulice-maven-plugin/src/it/**/*.java
+    est --dir=est --file=/dev/null
+    ./years.sh
```

Comment:
@krzyk I think we should set an exec attribute to this file
or use something like `cat ./years.sh|bash`